File: Labs/Lab 4/lab4.py
def reverseTree(self, root):
        if root is None:
            return

        temp = root.right
        root.right = root.left
        root.left = temp

        self.reverseTree(root.left)
        self.reverseTree(root.right)

    def invertTree(self, root):
        self.reverseTree(root)
        return root

File: Labs/Lab 4/lab4.java
public void reverseTree(TreeNode root) {
    if (root == null) {
        return;
    }

    TreeNode temp = root.right;
    root.right = root.left;
    root.left = temp;

    reverseTree(root.left);
    reverseTree(root.right);
}

public TreeNode invertTree(TreeNode root) {
    reverseTree(root);
    return root;
}
